pandas/tests/test_frame.py

```diff
@@ -8881,6 +8881,16 @@ def test_drop(self):
         df.drop(labels=df[df.b>0].index, inplace=True)
         assert_frame_equal(df,expected)
 
+        # multi-index issue
+        # origin of GH 11640
+        df = pd.DataFrame(columns=['a', 'b', 'c', 'd'],
+                          data=[[1, 'b1', 'c1', 3],
+                                [1, 'b2', 'c2', 4]])
+        dg = df.pivot_table(index='a', columns=['b', 'c'], values='d')
+        dg = dg.reset_index()
```

Comment:
I am confused here, I don't think this should work. ``dg.drop([('a','')],axis=1)`` works just fine. ``dg.drop('a',axis=1)`` is not specific enough & I don't think we level broadcast.